CHANGELOG.md

```
@@ -352,6 +361,8 @@ _📚 Refer to the [GitHub Release Notes](https://github.com/munich-quantum-tool
 
 <!-- PR links -->
 
+[#1662]: https://github.com/munich-quantum-toolkit/core/pull/1662
+[#1654]: https://github.com/munich-quantum-toolkit/core/pull/1654
```

markdownlint-cli2 (0.22.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "#1654"

(MD053, link-image-reference-definitions)